app/tests/commands/test_role.py

```diff
@@ -0,0 +1,408 @@
+
+    role.role_command(
+        ack,
+        {"text": "", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(help_text_fr)
+
+
+def test_role_command_handles_help_EN_command_EN_client():
+    # test that role command displays help text in English
+    ack = MagicMock()
+    respond = MagicMock()
+    client = MagicMock()
+    client.users_info.return_value = helper_client_locale()
+
+    role.role_command(
+        ack,
+        {"text": "help", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(help_text_en)
+
+
+def test_role_command_handles_help_EN_command_FR_client():
+    # test that the role command displays help text in French
+    ack = MagicMock()
+    respond = MagicMock()
+    client = MagicMock()
+    client.users_info.return_value = helper_client_locale()
+
+    role.role_command(
+        ack,
+        {"text": "help", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(help_text_en)
+
+
+def test_role_command_handles_help_FR_command_EN_client():
+    # test that the role command displays help text in French when the french help text is used
+    ack = MagicMock()
+    respond = MagicMock()
+    client = MagicMock()
+    client.users_info.return_value = helper_client_locale()
+
+    role.role_command(
+        ack,
+        {"text": "aide", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(help_text_fr)
+
+
+def test_role_command_handles_unknown_command_EN_client():
+    # test handling unknown command with English locale
+    ack = MagicMock()
+    respond = MagicMock()
+    client = MagicMock()
+    client.users_info.return_value = helper_client_locale()
+    role.role_command(
+        ack,
+        {"text": "foo", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(
+        "Sorry but I don't understand this command. Please type /role help to get usage information"
+    )
+
+
+def test_role_command_handles_unknown_command_FR_client():
+    # test handling unknown command with French locale
+    ack = MagicMock()
+    respond = MagicMock()
+    client = MagicMock()
+    client.users_info.return_value = helper_client_locale("fr")
+    role.role_command(
+        ack,
+        {"text": "foo", "command": "/role"},
+        MagicMock(),
+        respond,
+        client,
+        MagicMock(),
+    )
+    ack.assert_called
+    respond.assert_called_with(
+        "Désolé mais je ne comprends pas cette commande. Veuillez saisir /aide help pour obtenir des informations sur l'utilisation"
```

Comment:
Just realizing there's a "help" word in the middle of the French sentence. Wondering if the syntax shouldn't be reviewed.

When looking up the EN, we write:
`Please type /role help to get usage information`
Which means I'd probably update the FR to:
`Veuillez saisir /role aide pour obtenir des informations sur l'utilisation`

Reply:
This is corrected now. Thank you for noticing it.